java/src/jmri/util/JmriInsets.java

```diff
@@ -21,10 +21,7 @@
  * The standard insets command fails on Linux - this class attempts to rectify
  * that.
  * <p>
- * <a href="http://forums.sun.com/thread.jspa?threadID=5169228&start=29">
  * Borrows heavily from the Linsets class created by: A. Tres Finocchiaro
```

Comment:
I think it's this thread, if you are interested in keeping the link:
https://community.oracle.com/thread/1368416?start=29&tstart=0

Reply:
Thanks.